app/assets/javascripts/discourse/app/templates/tags/show.hbs

```diff
@@ -88,9 +88,11 @@
               {{topic-dismiss-buttons position="bottom" selectedTopics=selected
               model=model showResetNew=showResetNew showDismissRead=showDismissRead resetNew=(action "resetNew")}}
 
-              {{#footer-message education=footerEducation message=footerMessage}}
-                {{#link-to "tags"}} {{i18n "topic.browse_all_tags"}}{{/link-to}} {{i18n "or"}} {{#link-to "discovery.latest"}}{{i18n "topic.view_latest_topics"}}{{/link-to}}.
-              {{/footer-message}}
+              {{#unless list.canLoadMore}}
+                {{#footer-message education=footerEducation message=footerMessage}}
+                  {{#link-to "tags"}} {{i18n "topic.browse_all_tags"}}{{/link-to}} {{i18n "or"}} {{#link-to "discovery.latest"}}{{i18n "topic.view_latest_topics"}}{{/link-to}}.
```

Comment:
I know you didn't add this line but it bothers me that it's hard to translate, because the `or` is a separate token. I guess we can live with it a while longer.